src/main/fc/rc_controls.c

```diff
@@ -170,26 +178,32 @@ void processRcStickPositions(throttleStatus_e throttleStatus)
                 }
             }
         }
-    }
-
-    if (rcDelayCommand != 20) {
-        return;
-    }
-
-    if (isUsingSticksToArm) {
-        // Disarm on throttle down + yaw
+    } else if (rcDelayCommand == rcDelayTicks) {
         if (rcSticks == THR_LO + YAW_LO + PIT_CE + ROL_CE) {
+            // Disarm on throttle down + yaw
             if (ARMING_FLAG(ARMED))
                 disarm();
             else {
                 beeper(BEEPER_DISARM_REPEAT);    // sound tone while stick held
                 rcDelayCommand = 0;              // reset so disarm tone will repeat
             }
+        } else if (!ARMING_FLAG(ARMED)) {
+            if (rcSticks == THR_LO + YAW_HI + PIT_CE + ROL_CE) {
+                // Arm via YAW
+                tryArm();
+                return;
+            } else {
+                resetArmingDisabled();
+            }
         }
     }
 
-    if (ARMING_FLAG(ARMED)) {
-        // actions during armed
+    if (ARMING_FLAG(ARMED) || rcDelayCommand != rcDelayTicks / STICK_DELAY_RATIO) {
```

Comment:
Just add `rcDelayTicksArm` in addition to `rcDelayTicks` to get rid of this division.